recipes-kernel/packagegroups/packagegroup-ni-debug-kernel.bb

```diff
@@ -5,12 +5,11 @@ inherit packagegroup
 
 RDEPENDS_${PN} = "\
 	kernel-debug \
-	kernel-debug-base \
-	kernel-debug-vmlinux \
-	kernel-debug-image \
```

Comment:
Why are we dropping all these from the packagegroup? Are they deprecated?

Reply:
The packagegroup is over-specifying the dependencies. The dependency chain for _kernel-debug_ already includes the _kernel-debug-base_ dependency which in turn depends on the package providing _kernel-debug-image_. Here's the dependency chain reconstructed from .ipks I have built to test this:

```
  kernel-debug_4.14+git0+db94e1d70a-r0_x64.ipk
  Depends: kernel-debug-4.14.146-rt67-debug-dev-nilrt

	kernel-debug-4.14.146-rt67-debug-dev-nilrt_4.14+git0+db94e1d70a-r0_x64.ipk
	Depends: kernel-debug-image-4.14.146-rt67-debug-dev-nilrt
	Provides: kernel-debug-4.14.146-rt67-debug-dev-nilrt, kernel-debug-base

		kernel-debug-image-4.14.146-rt67-debug-dev-nilrt_4.14+git0+db94e1d70a-r0_x64.ipk
		Depends: kernel-debug-image-bzimage-4.14.146-rt67-debug-dev-nilrt
		Provides: kernel-debug-image

			kernel-debug-image-bzimage-4.14.146-rt67-debug-dev-nilrt_4.14+git0+db94e1d70a-r0_x64.ipk
			Provides: kernel-debug-image-bzimage

```
The removal of _kernel-debug-vmlinux_ is accidental and caused by the fact I was reusing the nohz/next package groups (which don't need/want the vmlinux binary). For the debug kernel the vmlinux package is needed bc. it contains the un-stripped kernel image with debug symbols. Thanks for catching this; will fix in V2.